consensus/core/src/blockstatus.rs

```diff
@@ -43,4 +43,8 @@ impl BlockStatus {
     pub fn is_valid(self) -> bool {
         self != BlockStatus::StatusInvalid
     }
+
+    pub fn is_invalid(self) -> bool {
+        self == BlockStatus::StatusInvalid
```

Comment:
Optional: seems strange that this isn't just `!is_valid()`

Reply:
I find it more readable